src/containers/SearchPage/components/results/SearchArticle.jsx

```diff
@@ -9,23 +9,27 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { toEditArticle } from '../../../util/routeHelpers';
-import { ArticleResultShape } from '../../../shapes';
-import { searchClasses } from '../SearchPage';
+import { Document } from 'ndla-icons/common';
+import { toEditArticle } from '../../../../util/routeHelpers';
+import { ArticleResultShape } from '../../../../shapes';
+import { searchClasses } from '../../SearchPage';
 
 const SearchArticle = ({ article, locale }) => (
   <div {...searchClasses('result')}>
+    <div {...searchClasses('image')}>
+      <img src="/assets/placeholder.png" alt="" />
+    </div>
     <div {...searchClasses('content')}>
       <Link
         {...searchClasses('link')}
         to={toEditArticle(article.id, article.articleType, locale)}>
-        <h1 {...searchClasses('title')}>{article.title}</h1>
+        <h2 {...searchClasses('title')}>
+          <Document />
```

Comment:
Maybe use the new icon?

Reply:
Foreslår at vi tar inn nye ikoner etter at https://github.com/NDLANO/editorial-frontend/pull/174 er tatt inn, da den har hjelpere mtp `resourceTypes` som må mappes her.